### PR TITLE
feat: allow adding extra ports to the application

### PR DESCRIPTION
**Pull request by omBratteng**
Adds the possibility to add more ports to the application service

### CODE REVIEW

**Review by idoshamun — COMMENTED** [review]
@omBratteng don't we need to add extra ports to the container as well?

**Comment by omBratteng** [comment]
Yes, probably 🙈

**Comment by idoshamun** [comment]
@omBratteng also I'd suggest a different approach. Allow providing the ports array and by default set it to port 80 like we do now

**Comment by omBratteng** [comment]
Good idea. Will change it 😄

**Comment by omBratteng** [comment]
@idoshamun marked the `port` as deprecated and added new `ports` which takes array of ports instead. Renamed `extraServicePorts` to `servicePorts` and default back to port 80 if no ports are given

File: src/k8s.ts [hunk 1]
```diff
@@ -379,6 +382,12 @@ export const createAutoscaledExposedApplication = ({
       (name) => `{"default": "${name}"}`,
     );
   }
+
+  const ports =
+    servicePorts && servicePorts.length > 0
```
Comment (idoshamun) [inline src/k8s.ts:387]:
```suggestion
    servicePorts?.length > 0
```

**Review by idoshamun — APPROVED** [review]
Perfect, please announce on #engineering once we release a new version

**Comment by idoshamun** [comment]
@omBratteng I see build fails, please fix it before merging

**Comment by omBratteng** [comment]
The build failed with your suggestion 🙈 But it should work... maybe just set default `servicePorts = []`?

**Pull request merged**